draft-ietf-tsvwg-careful-resume.xml

```diff
@@ -688,7 +689,7 @@ Observing ...> Connect -> Reconnaissance --------------------> Normal
             enters the Safe Retreat Phase
             (see trigger packet_loss and ECN_CE  in <xref target="sec-QLOG"></xref>).</t>
 
-            <t>Validating Phase (Receiving acknowledgement for all unvalidated packets):
+            <t>Validating Phase (Receiving acknowledgement for the unvalidated packets):
```

Comment:
Something seems weird with the 2 descriptions, could we make them both the same ("Acknowledgement of unvalidated packets"):

Validating Phase (Receiving acknowledgement for the unvalidated packets)
Safe Retreat Phase (Acknowledgement of unvalidated packets)